### ISSUE TITLE
ScriptSafe Firefox 57 under Windows: (Language) setting(s) either not saved or not loaded

### ISSUE THREAD

**Issue by Shalindor** [andryou/scriptsafe#295]
I changed the language from english to german and got the message "settings saved". After that I closed the firefox browser. Starting the firefox again and looking at ScriptSafe's little windows and/or settings page the language has been reverted to english. So either the setting is not really correctly saved or ScriptSafe does not correctly load the setting at start up.

**Comment by andryou** [comment]
Thanks for the report, I just made a commit to hopefully resolve this. I suspect it's a timing issue regarding setting initialization on start up.

**Comment by andryou** [comment]
Just wanted to ask: is everything else working? (e.g. the Options page is loading properly)

**Comment by Shalindor** [comment]
Yes, seems like anything else is working properly. So far the only thing i found was the
language setting bug, **which is still there even in 1.0.9.7**. Changing language is only temporary successful, after closing the browser completely and then start it next time, language is reverted to english.

**Comment by andryou** [comment]
Thank you! I hope to push a release out tomorrow which will more or less sync up the Chrome and Firefox builds of ScriptSafe. The Opera build is lagging as it is pending manual approval by the Opera Team.

**Comment by Shalindor** [comment]
Ok, updated today to 1.0.9.8. That fixed the language settings bug. Changing the language is now persistent even between completely new starts of the browser.
